File: Tugas coding modul 2/coding c/PRAK205-2310817120005-Noviana_Nur_Aisyah.c
#include <stdio.h>
#include <math.h>

void main(){
    float A, B;

    printf("Input\n");
    scanf("%f %f", &A, &B);

    float C = sqrt(B*B - A*A);
    float tinggi = A;
    float alas = C;
    float keliling = A + B + C;
    float luas = A * C / 2;

    printf("\nOutput\n");
    printf("Alas = %.0f cm\n", C);
    printf("Tinggi = %.0f cm\n", A);
    printf("Keliling = %.0f cm\n", keliling);
    printf("Luas = %.0f cm^2\n", luas);
}

File: Tugas coding modul 2/coding py/PRAK205-2310817120005-Noviana_Nur_Aisyah.py
import math

print('Input')
input_satu = input()
# 40 41 | 16 65

angka = input_satu.split()
A, B = map(int, angka) # A = tinggi, B = sisi miring

C = int(math.sqrt(B ** 2 - A ** 2)) # C = alas
keliling = int(A + B + C)
luas = int(1 / 2 * C * A)

print('\nOutput')
print(f'Alas = {C} cm')
print(f'Tinggi = {A} cm')
print(f'Keliling = {keliling} cm')
print(f'Luas = {luas} cm^2')